proposal.rst

```diff
@@ -267,29 +285,29 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-* Weeks of bonding: get familiar with ranking algorithms, search for suitable datasets, assessment of the complexity of integrating large databases, and performance tests.
 
-In each next position assumes that testing and writing cross-platform part is included.
+* Weeks of bonding: Get more familiar with ranking algorithms, select specific suitable parts of the above datasets, discuss what is wrong with `#278 <https://github.com/xapian/xapian/pull/278>`_.
 
-* 1 Week - integrate datasets 
+* 1 Week - FIRE dataset integration. Each user have to download it by themself, because of 
+registration required. Implement training process using Letor API.  
 
-* 2 Weeks of exams (situation with exams is very unstable now in my University, so I timeline may change in this case)
+* 2 Weeks of exams (situation with exams is very unstable now in my University, so I timeline may change in this case).
 
-* 1 Week - solve possible problems with the size of datasets
+* 1 Week - add function for selecting scorer and ranker. It would be nice to write API allows user to select default sets of queries to run and run specified queries. For that purpose update build system to be able to run suite from terminal.
```

Comment:
The evaluation suite can currently be run from the terminal, so I'm not sure what you're going to change here.

When you say "add function for selecting scorer and ranker", are you thinking something like the Registry we use for things like Weight classes, or something like `Weight::create()`? Most people writing code would just instantiate the classes they need, although in the evaluation suite there is a need for people to set configuration (somehow — and you should be clear how you want them to do this) saying which ranker(s) to evaluate.

Do you need people to select different scorers? Is there a reason not to run all of them and report the results?